runtime/handler.go

```diff
@@ -66,12 +67,16 @@ func ForwardResponseStream(ctx context.Context, mux *ServeMux, marshaler Marshal
 		case resp == nil:
 			buf, err = marshaler.Marshal(errorChunk(streamError(ctx, mux.streamErrorHandler, errEmptyResponse)))
 		default:
-			result := map[string]interface{}{"result": resp}
-			if rb, ok := resp.(responseBody); ok {
-				result["result"] = rb.XXX_ResponseBody()
+			if httpBody, ok := resp.(*httpbody.HttpBody); ok {
+				buf, err = marshaler.Marshal(httpBody)
```

Comment:
This shouldn't even need to consult the marshaler.
```suggestion
				buf = httpBody.GetData()
```

Reply:
Probably want to do something about the delimiter too.

Reply:
Please set the content type as well.

Reply:
will check on the content type.